### PR TITLE
Add service fee log

### PR DESCRIPTION
**Pull request by fhenneke**
This PR adds a log for the service fee charged to solvers in the COW DAO bonding pool.

This log is useful since it determines the budget for ongoing bounties.

This PR does not change the behavior of the code and no additional test was added.

### CODE REVIEW

File: src/fetch/payouts.py [hunk 1]
```diff
@@ -137,6 +137,15 @@ def reward_scaling(self) -> Fraction:
         The reward is multiplied by this factor"""
         return 1 - SERVICE_FEE_FACTOR * self.service_fee
 
+    def total_service_fee(self) -> Fraction:
+        """Scaling factor for service fee
+        The reward is multiplied by this factor"""
+        return (
+            SERVICE_FEE_FACTOR
+            * self.service_fee
+            * (self.primary_reward_cow + self.quote_reward_cow)
```
Comment (harisang) [inline src/fetch/payouts.py:146]: Can this be negative? If so, should we maybe redefine now how we measure the service fee collected?

File: src/fetch/payouts.py [hunk 2]
```diff
@@ -137,6 +137,15 @@ def reward_scaling(self) -> Fraction:
         The reward is multiplied by this factor"""
         return 1 - SERVICE_FEE_FACTOR * self.service_fee
 
+    def total_service_fee(self) -> Fraction:
+        """Scaling factor for service fee
```
Comment (harisang) [inline src/fetch/payouts.py]: Comment copy-pasted from above so probably needs editing

File: src/fetch/payouts.py (hunk 1 above)
Comment (fhenneke) [reply src/fetch/payouts.py:146]: It can be negative. I created a draft PR #424 for changing the behavior.

File: src/fetch/payouts.py (hunk 2 above)
Comment (fhenneke) [reply src/fetch/payouts.py]: https://github.com/cowprotocol/solver-rewards/pull/423/commits/7b725eecf67ea7331d6947cc5d0e45e93406e143

**Review by harisang — APPROVED** [review]

**Pull request merged**